gradle/verification-metadata.xml

```diff
@@ -82,22 +94,1282 @@
             <trusting group="org.apache.maven"/>
          </trusted-key>
          <trusted-key id="fa7929f83ad44c4590f6cc6815c71c0a4e0b8edd" group="net.java.dev.jna"/>
+         <trusted-key id="feb9209f2f2f3f4664841e55ac107b386692dadd" group="^software[.]amazon($|([.].*))" regex="true"/>
          <trusted-key id="ff6e2c001948c5f2f38b0cc385911f425ec61b51">
-            <trusting group="org.junit"/>
             <trusting group="junit"/>
+            <trusting group="org.junit"/>
          </trusted-key>
       </trusted-keys>
    </configuration>
    <components>
+      <component group="com.fasterxml" name="oss-parent" version="43">
```

Comment:
Please only add shas to things that are not signed, i spot checked and it seems like a large number of them have signatures. e.g. https://repo1.maven.org/maven2/software/amazon/awssdk/protocol-core/2.20.18/

Reply:
This is pretty much the output of `./gradlew --write-verification-metadata sha256 --export-keys` without any changes on my end, yea. What should I have done differently?

Reply:
Sadly a painful manual process. I would like to keep these minimal to be clear what is unsigned